pkg/proxy/ipvs/proxier.go

```diff
@@ -386,7 +386,7 @@ func NewProxier(ipt utiliptables.Interface,
 		return nil, fmt.Errorf("error parsing kernel version %q: %v", kernelVersionStr, err)
 	}
 	if kernelVersion.LessThan(version.MustParseGeneric(connReuseMinSupportedKernelVersion)) {
-		klog.Errorf("can't set sysctl %s, kernel version must be at least %s", sysctlConnReuse, connReuseMinSupportedKernelVersion)
+		klog.ErrorS(nil, fmt.Sprintf("can't set sysctl %s, kernel version must be at least %s", sysctlConnReuse, connReuseMinSupportedKernelVersion))
```

Comment:
Should we just leave this one as is? Seems odd to change to the structured log if there is no error to report in the structure.

I believe the KEP explicitly states that it is a non-goal to switch all log functions to the new ones.

Reply:
Firstly, we dont want to change the log level.  If error is nil, it will not print nil, instead log level will be info level.